FEXCore/include/FEXCore/IR/IR.h

```diff
@@ -536,6 +536,17 @@ enum IndexNamedVectorConstant : uint8_t {
   INDEXED_NAMED_VECTOR_MAX,
 };
 
+// This must directly match bytes to the named opsize.
+// Implicit sized IR operations does math to get between sizes.
+enum OpSize : uint8_t {
```

Comment:
Should this be an enum class?

Reply:
It will be once the implicit logic is completely gone. Since it gets converted to a u8 inside of the IR data member currently, that would have been a bigger structural undertaking that's easier to do after all the implicit logic is gone.